packages/tasks/src/hardware.ts

```diff
@@ -136,6 +136,10 @@ export const SKUS = {
 				tflops: 32.62,
 				memory: [24],
 			},
+			"V100": {
+				tflops: 130,
```

Comment:
We are using the float16 performance as a reference, as it's an usual workload for ML. In this case, we can estimate it to be about double the [single-precision in this table](https://images.nvidia.com/content/technologies/volta/pdf/volta-v100-datasheet-update-us-1165301-r5.pdf), or the 28.26 reported [here](https://www.techpowerup.com/gpu-specs/tesla-v100-pcie-16-gb.c2957).

```suggestion
				tflops: 28.26,
```